gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigResourceLocalHandler.java

```diff
@@ -305,4 +322,43 @@ public static FlowSpec createFlowSpecForConfig(FlowConfig flowConfig) {
       throw new FlowConfigLoggedException(HttpStatus.S_400_BAD_REQUEST, "bad URI " + flowConfig.getTemplateUris(), e);
     }
   }
+
+  protected String getErrorMessage(FlowSpec flowSpec) {
+    StringBuilder message = new StringBuilder("Flow was not compiled successfully.");
+    Map<String, ArrayList<String>> allErrors = new HashMap<>();
```

Comment:
FYI, I replaced `HashTable`, as that's a non-canonical collection (even surprising to see in java 1.8 code).  in fact, I first wondered whether there was a specific reason, perhaps due to json serialization... but AFAICT, the current unit tests exercise this function to thus validate that `HashMap` works just as well.